docs/source/concepts/local-ledger.rst

```diff
@@ -343,8 +344,14 @@ Local ledgers
 As explained in the Daml Ledger Model, parties see only a :ref:`projection <da-model-projections>` of the shared ledger for privacy reasons.
 Like consistency, projection extends to causality graphs as follows.
 
+Definition »Stakeholder informee«
+  A party `P` is a **stakeholder informee** of an action `act` if `P` is an informee of `act` and, if `act` is an action on a contract then `P` is a stakeholder of the contract.
```

Comment:
```suggestion
  A party `P` is a **stakeholder informee** of an action `act` on contract `c` if `P` is an informee of `act` and a stakeholder of `c`.
```

Reply:
That won't work for NoSuchKey actions because these actions don't act on a contract. I'll go with @phoebe-da 's suggestions of using bullet points.